ReportingServicesTools/Functions/CatalogItems/Get-RsSubscription.ps1

```diff
@@ -0,0 +1,141 @@
+                    $ActiveStateDataType = "$namespace.ActiveState"
+                    
+                    #Set ExtensionSettings
+                    $ExtensionSettings = New-Object $ExtensionSettingsDataType
+                    
+                    $ExtensionSettings.Extension = $subscription.DeliverySettings.Extension
+                    $ExtensionSettings.ParameterValues = $subscription.DeliverySettings.ParameterValues
+
+                    #Set ActiveState
+                    $ActiveState = New-Object $ActiveStateDataType
+
+                    $ActiveState.DeliveryExtensionRemoved          = $subscription.Active.DeliveryExtensionRemoved
+                    $ActiveState.DeliveryExtensionRemovedSpecified = $subscription.Active.DeliveryExtensionRemovedSpecified
+                    $ActiveState.SharedDataSourceRemoved           = $subscription.Active.SharedDataSourceRemoved
+                    $ActiveState.SharedDataSourceRemovedSpecified  = $subscription.Active.SharedDataSourceRemovedSpecified
+                    $ActiveState.MissingParameterValue             = $subscription.Active.MissingParameterValue
+                    $ActiveState.MissingParameterValueSpecified    = $subscription.Active.MissingParameterValueSpecified
+                    $ActiveState.InvalidParameterValue             = $subscription.Active.InvalidParameterValue
+                    $ActiveState.InvalidParameterValueSpecified    = $subscription.Active.InvalidParameterValueSpecified
+                    $ActiveState.UnknownReportParameter            = $subscription.Active.UnknownReportParameter
+                    $ActiveState.UnknownReportParameterSpecified   = $subscription.Active.UnknownReportParameterSpecified
+                    
+                    [pscustomobject]@{
+                        SubscriptionID        = $subscription.SubscriptionID
+                        Owner                 = $subscription.Owner
+                        Path                  = $subscription.Path
+                        VirtualPath           = $subscription.VirtualPath
+                        Report                = $subscription.Report
+                        DeliverySettings      = $ExtensionSettings
+                        Description           = $subscription.Description
+                        Status                = $subscription.Status
+                        Active                = $ActiveState
+                        LastExecuted          = $subscription.LastExecuted
+                        LastExecutedSpecified = $subscription.LastExecutedSpecified
+                        ModifiedBy            = $subscription.ModifiedBy
+                        ModifiedDate          = $subscription.ModifiedDate
+                        EventType             = $subscription.EventType
+                        IsDataDriven          = $subscription.IsDataDriven
+						MatchData             = $matchData
```

Comment:
please be consistent, except line 129 to 132 everything else use spaces instead of tabs